indico/modules/events/surveys/forms.py

```diff
@@ -115,3 +121,22 @@ class ImportQuestionnaireForm(IndicoForm):
     json_file = FileField(_('File'), accepted_file_types="application/json,.json",
                           description=_("Choose a previously exported survey content to import. "
                                         "Existing sections will be preserved."))
+
+
+class EmailPeopleForm(IndicoForm):
```

Comment:
I'd call this `InvitationForm` (we also called it like this in the registration module when inviting people)